platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBeanMBean.java

```diff
@@ -20,87 +20,95 @@
  * Interface for the Application Service MBean. Allows exposing the application service out via a
  * JMX MBean interface.
  */
+@Deprecated
 public interface ApplicationServiceBeanMBean {
 
-  /**
-   * Creates an application hierarchy tree that shows relationships between applications.
-   *
-   * @return A list of the root applications expressed as maps.
-   */
-  List<Map<String, Object>> getApplicationTree();
-
   /**
    * Creates an application list that has two attributes that describes relationships between
    * applications (parent and children dependencies).
    *
    * @return A list of the root applications expressed as maps.
    */
+  @Deprecated
   List<Map<String, Object>> getApplications();
 
   /**
-   * Starts an application with the given name.
+   * Gets all installation profiles on the system.
    *
-   * @param appName Name of the application to start.
-   * @return true if the application was successfully started, false if not.
+   * @return installation profile objects.
    */
-  boolean startApplication(String appName);
+  @Deprecated
+  List<Map<String, Object>> getInstallationProfiles();
 
   /**
-   * Stops an application with the given name.
+   * TODO:
```

Comment:
:question: What is the TODO?

Reply:
This is an existing TODO comment for the getServices method, none of the javadoc was changed in this class. Not sure why this is marked as an addition.